### ISSUE TITLE
Player Spiders Feedback/bugs/playability

### ISSUE THREAD

**Issue by Niknakflak** [tgstation/-tg-station#12734]
1. I don't really know if it makes sense for spiders to have nightvision but when player spiders are used as an alternative to xenos and they can't see shit in maintenance, they need night vision.
2. They should be able to lay web down without costing "hunger" or whatever it is that requires them to do that.  I'm not sure much about the details about this except that it didn't work super well.
3. Infertility makes it hard to make like a spider queen be a continues baby maker if they suddenly stop making babies for some reason.  I also don't know if this is a real thing because I'm only going off what players told me and not complete tests of my own.

tl;dr player spiders are kinda crappy and need some rework.  There's not point having the feature if they are basically unplayable in it.

**Comment by Incoming5643** [comment]
when I made them playable it wasn't a full feature thing, I was just laying the groundwork for fleshing them out down the road if someone wanted to.

There's no hunger or infertility mechanics. The only thing limited is egg laying. You can lay one set of eggs for every human you eat. Eating a human happens automatically once you wrap a human as a green spider.

Also everyone knows this but since one set of eggs from a player spider makes like 8 more player spiders these things are hilariously overpowered en mass and will also aggressively empty ghost chat for players.

**Comment by Zilenan91** [comment]
It's 5 more spiders. But yeah currently spiders are bugged in that if you start wrapping or doing any other action like laying webs or whatever, and then it gets interrupted either by you moving or being bumped, you cannot ever do that action ever again unless you're killed and admin revived. This makes them more than a little bit unplayable.

**Comment by Incoming5643** [comment]
I swear I saw a bug fix omnibus pull that fixed that issue.

Also splitting hairs here, but spiders can't actually get more spiders from mice, as they turn into items on death.

**Comment by Zilenan91** [comment]
Yeah you can wrap them when alive. Spiders can actually wrap anything while alive and re-use them. An example is biting a dude as a nurse and letting him drop from the morphine. Once he's down and not dead, wrap and lay, and he'll wake up, and at that point you can keep biting him and letting him drop from the morphine forever, continually getting more spiders.

So wrap the mice while alive.

**Comment by Shadowlight213** [comment]
@zilenan91 surviving a wrapping is a bug. I'm fairly certain that being wrapped up by a spider is supposed to instantly kill you like being absorbed by a ling as you're having your fluids sucked out.

**Comment by Incoming5643** [comment]
Actually it's not, you can even resist out of being wrapped.

**Comment by Zilenan91** [comment]
That's the bug part @Incoming5643 , that it doesn't instantly kill you. Spiders are kind of a buggy mess.

**Comment by Dorsisdwarf** [comment]
https://github.com/tgstation/-tg-station/blob/afde4474818cf6a21d4aed0b6bcfd30de9d41f3c/code/modules/mob/living/simple_animal/hostile/giant_spider.dm#L168

Nothing in there whatsoever about "instantly killing you"
